server/pulp/server/content/web/settings.py

```diff
@@ -1,5 +1,45 @@
+import ConfigParser
+import logging
 import os
 
+from pulp.server import logs as pulp_logs
+from pulp.server import config
+
+
+try:
+    log_level = config.config.get('server', 'log_level')
+    log_level = getattr(logging, log_level.upper())
+except (ConfigParser.NoOptionError, AttributeError):
+    # If the user didn't provide a log level, or if they provided an invalid one, let's use the
+    # default log level
+    log_level = pulp_logs.DEFAULT_LOG_LEVEL
+
+LOGGING = {
```

Comment:
It does mimic the behaviour of `start_logging`. The reason I chose to use Django's facilities is that it handles actually setting up the logs, where-as if I wanted to use `start_logging` as far as I could tell I would need to make use of `pulp.server.webservices.application` or create something similar, and things got more complicated from there.

My gut told me using Django to do this was the "right" way to do it, but I'm far from an expert on this.